src/Extractors/Csv.php

```diff
@@ -118,7 +118,8 @@ protected function makeRow(array $row, array $columns): array
         foreach ($columns as $column => $index) {
             // The is bool is redundant, it is necessary because of PHP Stan, since we check the type in initialize()
             if ((bool) $this->throwError && false === array_key_exists($index - 1, $row)) {
-                throw new InvalidInputException("Row with index #{$this->currentRow} does not have the '{$column}' field.");
+                $message = "Row with index #{$this->currentRow} does not have the '{$column}' field.";
+                throw new InvalidInputException($message);
```

Comment:
Would this be ok with phpcs ?
```suggestion
                throw new InvalidInputException(
                    "Row with index #{$this->currentRow} does not have the '{$column}' field."
                );
```

Reply:
It is OK with PHPCS PSR scan, but it fails the PHP CS Fixer spec that you have included .php_cs.dist